templates/app_inactive_401.html

```diff
@@ -19,7 +19,7 @@
     </style>
 </head>
 <body>
-    <p class="Normal404Text">403 - Forbidden</p>
+    <p class="Normal404Text">401 - Forbidden</p>
```

Comment:
This works as a starting point, but there's some confusion in this file, in the head element we have 401 Unauthorized, then down here we have 401 Forbidden (Forbidden is usually 403), and the class used for the formatting says 404. We should get better consistency across these elements, probably sticking with 401 Unauthorized.